prow/config.yaml

```diff
@@ -933,6 +933,78 @@ periodics:
         path: /mnt/disks/ssd0
       name: cache-ssd
 
+- name: ci-kubernetes-e2e-gce-ubuntu-node
+  interval: 30m
+  spec:
+    containers:
+    - args:
+      - --timeout=50
+      - --bare
+      - --gce-ssh=/etc/ssh-key-secret/ssh-private
+      - --gce-pub=/etc/ssh-key-secret/ssh-public
+      env:
+      - name: GOOGLE_APPLICATION_CREDENTIALS
+        value: /etc/service-account/service-account.json
+      - name: USER
+        value: prow
+      image: gcr.io/k8s-testimages/kubekins-e2e-prow:v20170418-c08e1094
+      volumeMounts:
+      - mountPath: /etc/service-account
+        name: service
+        readOnly: true
+      - mountPath: /etc/ssh-key-secret
+        name: ssh
+        readOnly: true
+      - mountPath: /root/.cache
+        name: cache-ssd
+    volumes:
+    - name: service
+      secret:
+        secretName: service-account
+    - name: ssh
+      secret:
+        defaultMode: 256
+        secretName: ssh-key-secret
+    - hostPath:
+        path: /mnt/disks/ssd0
+      name: cache-ssd
+
+- name: ci-kubernetes-e2e-gce-ubuntu-node-serial
+  interval: 30m
+  spec:
+    containers:
+    - args:
+      - --timeout=180
+      - --bare
+      - --gce-ssh=/etc/ssh-key-secret/ssh-private
+      - --gce-pub=/etc/ssh-key-secret/ssh-public
```

Comment:
Wait... the node job runs docker commands inside `e2e-node-jenkins.sh`, so making the scenario runs locally without docker will not solve the issue since the job still runs docker commands...

@fejta @spxtr thoughts?

@yguo0905 guess put the job to https://github.com/kubernetes/test-infra/blob/master/jenkins/job-configs/kubernetes-jenkins/bootstrap-ci-repo.yaml for now.